### PR TITLE
savegame fixes

### LINKED ISSUE #1375: Lifts break after reloading
When I save shortly after activating a lift and the lift is still down and then reload the save, it will stop working and can cause me to get softlocked. 
Happened on scythe 2, dbp 09 and doom 2.
Commit:  dd4e2b5

### PR DESCRIPTION
**Pull request by rfomin**
Fix #1375, #1374

### CODE REVIEW

File: src/p_saveg.c [hunk 1]
```diff
@@ -1232,7 +1224,16 @@ static void saveg_read_vldoor_t(vldoor_t *str)
     str->topcountdown = saveg_read32();
 
     // line_t *line;
```
Comment (fabiangreffrath) [inline src/p_saveg.c:1226]: Can we get rid of `readp()` altogether?

Comment (rfomin) [reply src/p_saveg.c:1226]: Yes, we can explicitly skip pointers. But sometimes `NULL` is written with checks in other places, we have to find them all.

I suggest we hold off on this for now.

Comment (fabiangreffrath) [reply src/p_saveg.c:1226]: Yes, okay.

**Review by fabiangreffrath — APPROVED** [review]
Thank you!

**Pull request merged**